sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -95,26 +95,45 @@ export class KeyVaultAccessControlClient {
       ? challengeBasedAuthenticationPolicy(credential)
       : signingPolicy(credential);
 
-    const internalPipelineOptions = {
-      ...pipelineOptions,
-      ...{
-        loggingOptions: {
-          logger: logger.info,
-          logPolicyOptions: {
-            allowedHeaderNames: [
-              "x-ms-keyvault-region",
-              "x-ms-keyvault-network-info",
-              "x-ms-keyvault-service-version"
-            ]
-          }
-        }
+    const internalPipelineOptions: InternalPipelineOptions = {
+      // coreHttp.PipelineOptions has "serviceVersion", but InternalPipelineOptions doesn't. Is that expected?
```

Comment:
I don't see `serviceVersion` on `PipelineOptions` - isn't that added by the service interface that extends `PipelineOptions`?

Reply:
I removed this property! Thank you